tests/unit/schemas/test_strategic_caps.py

```
@@ -0,0 +1,329 @@
+            "confidence": 0.7,
+        },
+        "swot": {
+            "strengths": ["Brand loyalty", "Balance sheet strength"],
+            "weaknesses": ["R&D underinvestment"],
+            "opportunities": ["Emerging markets"],
+            "threats": ["Tariffs"],
+            "strategic_assessment": "Solid moat, needs to reinvest.",
+            "strategic_score": 0.6,
+            "confidence": 0.65,
+        },
+    }
```

Ruff (0.16.1)

[warning] Mutable default value for class attribute

(RUF012)